src/instruments/src/EFB/README.md

```diff
@@ -6,7 +6,11 @@ You can run a standalone version of the EFB by running the following:
 
 ```
 $ npm install
-$ npx webpack serve
+$ npx webpack serve -c webpack.development.config.js
 ```
 
-Then, navigate to http://localhost:8080.
\ No newline at end of file
+Then, navigate to http://localhost:8080.
+
+The native resolution of the EFB in the sim is 1400x1000. 
```

Comment:
1430x1000

Reply:
could you help me verify this? I copied this resolution from the discord where someone said it was 1400x1000, but I'm happy to change this if it's wrong

Reply:
ok I found it, will correct this